### DISCUSSION TITLE
Neumorphic Shore

### DISCUSSION THREAD

**Discussion by Beingpax** [Akifyss/obsidian-border#161] [Show and tell]
Just created this with the help of chatGPT. It's a combination of Neumorphisma and Shore preset. 

`{
  "Appearance-light@@accent-light": "#19C8BC",
  "Appearance-light@@accent-color-override-light": true,
  "Appearance-light@@mod-left-split-background-select-light": "mod-left-split-background-CSS-light",
  "Appearance-light@@background-mod-left-CSS-light": "rgba(255, 255, 255, 0.8)",
  "Appearance-light@@mod-right-split-background-select-light": "mod-right-split-background-CSS-light",
  "Appearance-light@@background-mod-right-CSS-light": "rgba(255, 255, 255, 0.8)",
  "Appearance-light@@mod-root-split-background-select-light": "mod-root-split-background-CSS-light",
  "Appearance-light@@background-mod-root-CSS-light": "rgba(255, 255, 255, 0.9)",
  "Appearance-light@@background-mod-root-CSS-backdrop-filter-light": "blur(5px)",
  "Appearance-light@@background-underlying-select-light": "background-underlying-CSS-light",
  "Appearance-light@@background-underlying-CSS-light": "url(\"data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg'",
  "Appearance-light@@background-underlying-CSS-blend-mode-light": "normal",
  "Appearance-light@@on-border-light": "rgba(255, 255, 255, 0.9)",
  "Appearance-light@@card-shadow-light": "inset -4px -4px 10px 0 rgba(255, 255, 255, 0.6), inset 4px 4px 10px 0 rgba(116, 125, 136, .5)",
  "Appearance-light@@color-to-tab-icon-light": true,
  "Appearance-light@@background-activated-tab-header-light": "rgba(255, 255, 255, 0.8)",
  "Appearance-light@@shadow-activated-tab-header-light": "inset -4px -4px 6px 0 rgba(255, 255, 255, 0.5), inset 4px 4px 6px 0 rgba(116, 125, 136, .3)"
}`
![SCR-20240203-ondr](https://github.com/Akifyss/obsidian-border/assets/101010368/e1b0bdcb-4ea9-47df-af55-3f8c8f021806)

**Comment by Akifyss** [comment]
Great job!!

**Comment by zayKenyon** [comment]
> Just created this with the help of chatGPT. It's a combination of Neumorphisma and Shore preset.
> 
> `{ "Appearance-light@@accent-light": "#19C8BC", "Appearance-light@@accent-color-override-light": true, "Appearance-light@@mod-left-split-background-select-light": "mod-left-split-background-CSS-light", "Appearance-light@@background-mod-left-CSS-light": "rgba(255, 255, 255, 0.8)", "Appearance-light@@mod-right-split-background-select-light": "mod-right-split-background-CSS-light", "Appearance-light@@background-mod-right-CSS-light": "rgba(255, 255, 255, 0.8)", "Appearance-light@@mod-root-split-background-select-light": "mod-root-split-background-CSS-light", "Appearance-light@@background-mod-root-CSS-light": "rgba(255, 255, 255, 0.9)", "Appearance-light@@background-mod-root-CSS-backdrop-filter-light": "blur(5px)", "Appearance-light@@background-underlying-select-light": "background-underlying-CSS-light", "Appearance-light@@background-underlying-CSS-light": "url(\"data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg'", "Appearance-light@@background-underlying-CSS-blend-mode-light": "normal", "Appearance-light@@on-border-light": "rgba(255, 255, 255, 0.9)", "Appearance-light@@card-shadow-light": "inset -4px -4px 10px 0 rgba(255, 255, 255, 0.6), inset 4px 4px 10px 0 rgba(116, 125, 136, .5)", "Appearance-light@@color-to-tab-icon-light": true, "Appearance-light@@background-activated-tab-header-light": "rgba(255, 255, 255, 0.8)", "Appearance-light@@shadow-activated-tab-header-light": "inset -4px -4px 6px 0 rgba(255, 255, 255, 0.5), inset 4px 4px 6px 0 rgba(116, 125, 136, .3)" }` ![SCR-20240203-ondr](https://private-user-images.githubusercontent.com/101010368/302030370-e1b0bdcb-4ea9-47df-af55-3f8c8f021806.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3MTQyMTEsIm5iZiI6MTcxMDcxMzkxMSwicGF0aCI6Ii8xMDEwMTAzNjgvMzAyMDMwMzcwLWUxYjBiZGNiLTRlYTktNDdkZi1hZjU1LTNmOGM4ZjAyMTgwNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxN1QyMjE4MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZjkxODliOWM5NTgxYWZhOGRkNDk1N2RiZjM2NTAzY2JiYWU3NWM4ODY5ZmZkZTJhZGRhYmNhYjhjNTMwY2Y3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.2goAsrHcetT7qzWHBw1aV87llP8r-NiXCX19qxo9a5I)

Apologies for being off-topic, I looked at your profile and couldn't find a different way to contact you @Beingpax , what is that Task Card in the bottom right corner of your screen? It looks really interesting and I'd love to take a look at that plugin if you wouldn't mind 😆

**Reply by PNRYates** [reply]
its [taskido](https://github.com/702573N/Obsidian-Tasks-Timeline?tab=readme-ov-file#quick-entry-panel)
